src/platform/forms-system/src/js/components/PreSubmitSection.jsx

```diff
@@ -27,10 +28,27 @@ export function PreSubmitSection({
       )}
     </div>
   );
+
+  return (
+    <div>
```

Comment:
This is that conditional override I added so we can pass in customComponents, not sure of a better way to do this, probably need to refactor this so we can pass custom review components in the future.

Reply:
I think most forms that need to modify some part of the form system make a copy of the component in their form folder and modify it there... I'm not sure what the best solution would be.

Reply:
:thinking: I think it doesn't make sense to have `preSubmitInfo` to have the options to pass to the default `PrivacyAgreement` _and_ pass a custom component.

I recommend
1. Overriding the entire `PreSubmitSection` in the [`SubmitController`](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms-system/src/js/review/SubmitController.jsx#L90-L95) if `formConfig.preSubmitSection` exists
2. Passing the `preSubmitInfo` as a prop to the custom component
3. Doing this all in a separate PR; this one is too gnarly to include platform changes